File: Domain/Entities/PersonInContent.cs
namespace Domain.Entities;

public class PersonInContent
{
    public int Id { get; set; }
    
    public long ContentId { get; set; }
    public ContentBase Content { get; set; } = null!;

    public string Name { get; set; } = null!;

    public int ProfessionId { get; set; }
    public Profession Profession { get; set; } = null!;
}

File: Mobile/netflix/lib/domain/models/content/person_in_content.dart
import 'package:netflix/domain/models/content/profession.dart';

class PersonInContent {
  final int id;
  final int contentId;
  final String name;
  final Profession profession;

  const PersonInContent({required this.id, required this.contentId, required this.name, required this.profession});
}